docs/contributing/index.rst

```diff
@@ -40,52 +40,82 @@ Preperations for a new Python version
+              self.not_allowed(node)
 
-Add a corresponding changelog entry.
+      Please note, that for all AST Nodes without an explicit ``visit_<AST Node>`` method, a default applies which denies the usage of this expression and functionality.
+      As we try to be **as explicit as possible**, all language features should have a corresponding ``visit_<AST Node>``.
+      That follows the Zen of Python:
 
-Additionally modify ``.meta.toml`` and run the ``meta/config`` script to update the following files:
+      .. code-block:: python
+          :emphasize-lines: 5
 
-* ``/setup.py`` - Check that the new Python version classifier has been added ``"Programming Language :: Python :: <version>",``, and that the ``python_requires`` section has been updated correctly.
-* ``/tox.ini`` - Check that a ``testenv`` entry is added to the general ``envlist`` statement.
-* ``/.github/workflows/tests.yml`` - Check that a corresponding Python version entry has been added to the matrix definition.
+          >>> import this
+          The Zen of Python, by Tim Peters
 
-Run the tests via ``tox``.
+          Beautiful is better than ugly.
+          Explicit is better than implicit.
+          Simple is better than complex.
+          Complex is better than complicated.
+          Flat is better than nested.
+          Sparse is better than dense.
+          Readability counts.
+          Special cases aren't special enough to break the rules.
+          Although practicality beats purity.
+          Errors should never pass silently.
+          Unless explicitly silenced.
+          In the face of ambiguity, refuse the temptation to guess.
+          There should be one-- and preferably only one --obvious way to do it.
+          Although that way may not be obvious at first unless you're Dutch.
+          Now is better than never.
+          Although never is often better than *right* now.
+          If the implementation is hard to explain, it's a bad idea.
+          If the implementation is easy to explain, it may be a good idea.
+          Namespaces are one honking great idea -- let's do more of those!
 
-Create a pull request.
+  * Add a corresponding changelog entry.
+  * Additionally modify ``.meta.toml`` and run the ``meta/config`` script (for details see: https://github.com/mgedmin/check-python-versions) to update the following files:
+
+    * ``/setup.py`` - Check that the new Python version classifier has been added ``"Programming Language :: Python :: <version>",``, and that the ``python_requires`` section has been updated correctly.
+    * ``/tox.ini`` - Check that a ``testenv`` entry is added to the general ``envlist`` statement.
+    * ``/.github/workflows/tests.yml`` - Check that a corresponding Python version entry has been added to the matrix definition.
+    * ``/docs/conf.py`` - Add Python version to ``intersphinx_mapping`` list
+
+  * Run the tests via ``tox`` locally before push.
```

Comment:
Move the step to create a pull request here. I would like to see a preview of the final version, to make sure everything is nested appropriately. It's hard to tell when there are only two spaces of indentation, instead of 4 spaces, for example.

```rst
-   Item 1
-   Item 2

    -   Item 2.1
    -   Item 2.2

        -   Item 2.2.1
        -   Item 2.2.2

    -   Item 2.3

-   Item 3
```

This is also consistent with Python code blocks, so you never have to worry about how many spaces are needed. The extra lines may be required between different levels of nesting, else errors may be thrown. Yeah, rst can be finicky.